conf/application.conf

```diff
@@ -74,6 +74,11 @@ microservice {
       host = localhost
       port = 9218
     }
```

Comment:
can this config `ttp` be removed?

Reply:
not yet, but it will in the next ticket. Eligibility has it's own microservice and different routes, annoyhingly